usmqe/web/tendrl/mainpage/clusters/cluster_list/pages.py

```diff
@@ -32,34 +68,81 @@ def start_create_cluster(self):
         """
         self._model.create_btn.click()
 
-    def import_cluster(self, cluster_id=None, name=None, hosts=None):
+    def import_cluster(self, cluster_ident=None, name=None, hosts=None):
         """
         import cluster
 
         Parameters:
-            cluster_id (str): cluster id for choosing a cluster
-                              which should be imported
+            cluster_ident: cluster identificator for choosing a cluster
+                           which should be imported
```

Comment:
yes and no, I am using it for different item, so the reason is to not confuse one for other